django/db/migrations/state.py

```diff
@@ -226,6 +225,17 @@ def __init__(self, real_apps, models, ignore_swappable=False):
             labels = (".".join(model_key) for model_key in self._pending_operations)
             raise ValueError(msg % ", ".join(labels))
 
+    def bulk_update(self):
+        self._ready = self.ready
```

Comment:
is there a need to store `self._ready` on self instead of simply using a local variable (besides for checking it in tests)? `del self._ready` seems a bit ugly.